### DISCUSSION TITLE
i18n for readme files

### DISCUSSION THREAD

**Discussion by steve02081504** [community/community#50719] [Repositories]
### Select Topic Area

Product Feedback

### Body

You can see that many repo's have not only `README.md`, but also [`README_EN.md`](https://github.com/search?q=README_EN.md&type=code) [`README_CN.md`](https://github.com/search?q=README_CN.md&type=code) and [`README.jp.md`](https://github.com/search?q=README.jp.md&type=code), and [`/docs/README.zh-cn.md`](https://github.com/search?q=README.zh-cn.md&type=code) etc.
These files were created to make the repo easier to understand in different languages, but instead of automatically selecting the appropriate readme file to display based on the viewer's browser language setting, github only pickup `README.md` or `README.txt` which in my point of view is imcomplete and can be improved.
I wish github would improve this aspect of performance and make localisation of repo's easier.

**Comment by HG651454** [comment]
Nice work, deez very helpful.

**Reply by cbarney813** [reply]
A language isolation option would be helpful if there isn't already one.
